src/binarySearchTree.js

```diff
@@ -0,0 +1,95 @@
+import BinaryNode from './binarynode'
+
+export default class BinarySearchTree {
+  constructor() {
+    this.length = 0
```

Comment:
Consider making `length` a more meaningful identifier, like `size` - this was a little confusing in the context of a BST.